content/open-source/concepts/next-js/next-js.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'Next.js'
+Description: 'Next.js is an open-source framework that brings the best of React client-side development to the server-side along with built-in features like TypeScript support and route pre-fetching.'
+Subjects:
+  - 'Developer Tools'
+  - 'Web Development'
+Tags:
+  - 'Node'
+  - 'Open Source'
+  - 'React'
+CatalogContent:
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/computer-science'
+---
+
+[**Next.js**](https://nextjs.org) is an open-source development [framework](https://www.codecademy.com/resources/docs/general/framework) that brings the best of [React's](https://www.codecademy.com/resources/docs/react) client-side functionality to the [server-side](https://www.codecademy.com/resources/docs/general/server-side-rendering) when building web applicatons.
+
+Additionally, Next.js comes with built-in features including the following:
+
+- A page-based [routing](https://www.codecademy.com/resources/docs/general/routing) system that supports dynamic routes.
+- Every page is pre-rendered for improved site performance and search-engine optimization (SEO).
+- Any page can either be statically generated (SSG) ahead of time or rendered on the server-side (SSR) upon request.
```

Comment:
SSG is static-site generator and SSR is server-side rendering

Is it okay to use these acronyms when these terms aren't actually being used?

Reply:
I see your point, these acronyms seem rather forced in the text. Let's omit them.